dev-guide/api-conventions.md

```diff
@@ -430,3 +459,17 @@ API to meet the latest conventions.
 
 When adhered to the conventions prevent us from making API design mistakes or repeating them.
 As such, the existence of non-compliant APIs is not a justification for introducing additional non-compliant APIs.
+
+### I'm copying an API from an upstream project or platform/cloud provider, can I change the design?
+
+Yes! When designing an API for an abstraction, you should consider the end user experience and the value your
+abstraction is providing. If you copy the API verbatim, are you adding any value?
+
+If an upstream/platform API is not intuitive, we can improve the user experience by creating our own naming that better
+describes the effects of enabling a specific field with a specific value. Think about why a user would configure a field
+when choosing the name.
+
+When writing APIs for OpenShift, we try to make our APIs consistent with one another and "Kube-like" so that users of
+OpenShift have an understanding of how to configure our APIs intuitively. If an upstream API is not consistent with
```

Comment:
"configure APIs" looks a bit cryptic to me to be honest... Perhaps "interact with" or "use" would be more specific.